### PR TITLE
Doc : Modify the attribute description

### PR DESCRIPTION
**Pull request by q977734161**
Some Table behavior properties attribute descriptions are a bit problematic

### CODE REVIEW

File: site/docs/configuration.md [hunk 1]
```diff
@@ -60,11 +60,11 @@ Iceberg tables support table properties to configure table behavior, like the de
...
 | commit.retry.min-wait-ms           | 100              | Minimum time in milliseconds to wait before retrying a commit |
 | commit.retry.max-wait-ms           | 60000 (1 min)    | Maximum time in milliseconds to wait before retrying a commit |
-| commit.retry.total-timeout-ms      | 1800000 (30 min) | Maximum time in milliseconds to wait before retrying a commit |
+| commit.retry.total-timeout-ms      | 1800000 (30 min) | Totals retry timeout period in milliseconds for a commit |
 | commit.status-check.num-retries    | 3                | Number of times to check whether a commit succeeded after a connection is lost before failing due to an unknown commit state |
 | commit.status-check.min-wait-ms    | 1000 (1s)        | Minimum time in milliseconds to wait before retrying a status-check |
 | commit.status-check.max-wait-ms    | 60000 (1 min)    | Maximum time in milliseconds to wait before retrying a status-check |
-| commit.status-check.total-timeout-ms| 1800000 (30 min) | Maximum time in milliseconds to wait before retrying a status-check |
+| commit.status-check.total-timeout-ms| 1800000 (30 min) | Totals status-check timeout period in milliseconds |
```
Comment (kbendick) [inline site/docs/configuration.md:67]: Nit / non-blocking: Consider `Total timeout period in which the commit status-check must succeed, in milliseconds`

Also, `Total` has an extra `s` presently here as well.

**Review by kbendick — APPROVED** [review]
These are good finds @q977734161!

Some small feedback. But overall, this is a good discovery!

File: site/docs/configuration.md [hunk 2]
```diff
@@ -60,11 +60,11 @@ Iceberg tables support table properties to configure table behavior, like the de
 | commit.retry.num-retries           | 4                | Number of times to retry a commit before failing              |
 | commit.retry.min-wait-ms           | 100              | Minimum time in milliseconds to wait before retrying a commit |
 | commit.retry.max-wait-ms           | 60000 (1 min)    | Maximum time in milliseconds to wait before retrying a commit |
-| commit.retry.total-timeout-ms      | 1800000 (30 min) | Maximum time in milliseconds to wait before retrying a commit |
+| commit.retry.total-timeout-ms      | 1800000 (30 min) | Totals retry timeout period in milliseconds for a commit |
```
[the remark this answers is not in the document]
Comment (q977734161) [reply site/docs/configuration.md]: Tks @kbendick ,I have modified this.

File: site/docs/configuration.md (hunk 1 above)
Comment (q977734161) [reply site/docs/configuration.md]: Tks @kbendick ,I have modified this

**Review by rdblue — APPROVED** [review]

**Comment by rdblue** [comment]
Thanks, @q977734161!

**Pull request merged**